### PR TITLE
[MIG] datamodel: Migration to 16.0

### PR DESCRIPTION
**Pull request by aktiv-heli-kantawala**

### CODE REVIEW

File: datamodel/builder.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class DatamodelBuilder(models.AbstractModel):
     _datamodels_registry_cache_size = DEFAULT_CACHE_SIZE
 
     def _register_hook(self):
+
```
Comment (lmignon) [inline datamodel/builder.py]: is this blank line required?

Comment (aktiv-heli-kantawala) [reply datamodel/builder.py]: Fixed. Please, review it.

**Comment by StefanRijnhart** [comment]
FYI this module was already included in https://github.com/OCA/rest-framework/pull/313, but this PR can be reviewed and merge separately of course as it has no dependencies. Of course, in #313 the module is actually set to installable :stuck_out_tongue:

**Comment by lmignon** [comment]
@heliaktiv Can you review #313 where @StefanRijnhart already did the migration. I'll close this one as duplicate of #313

**Pull request closed without merging**